contracts/Vault.vy

```diff
@@ -1735,21 +1713,18 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
     # Check report is within healty ranges
 
     if self.strategies[msg.sender].enforceChangeLimit:
```

Comment:
`if self.healthCheck != ZERO_ADDRESS && HealthCheck(self.healthCheck).doHealthCheck(msg.sender)`

removes the struct modification and delegates the disabling of one time healthcheck to the contract.

Also not ideal but we can entirely disable healthchecks for a vault if there is a problem as a nuclear option by setting to 0x0.